packages/components/src/util/clickable.module.css

```diff
@@ -0,0 +1,164 @@
+.button {
+  /* Base Styles */
+  @apply rounded border;
+
+  /* Base Text Styles */
+  @apply font-bold text-sm leading-body;
+}
+
+/* Component Sizes */
+.button,
+.button--size-small {
+  @apply px-5 py-2 text-sm leading-snug;
+}
+
+.button--size-medium {
+  @apply px-6 py-4 text-sm leading-snug;
+}
+
+.button--size-large {
+  @apply px-8 py-5 leading-7;
+
+  /* manual font size sinde there's no match in tokens */
+  font-size: 1.25rem;
```

Comment:
not blocking, but in its final form this should definitely have a corresponding token size